docs/application/vstools-mac/overview.md

```diff
@@ -1,5 +1,8 @@
 # Visual Studio for Mac Extension for Tizen
 
+> [!NOTE]
+> There are no further distribution plan for Visual Studio for Mac Extension for Tizen after Oct. 29, 2021 release (version 1.6).

```

Comment:
Since ‘distribution plan’ is a singular noun, ‘is’ should be used instead of ‘are’.
```suggestion
> There is no further distribution plan for Visual Studio for Mac Extension for Tizen after Oct. 29, 2021 release (version 1.6).

```